common/src/main/java/com/microsoft/hydralab/common/appcenter/entity/HandledErrorLog.java

```diff
@@ -17,7 +17,7 @@ public class HandledErrorLog implements Log {
     /**
      * Log type.
      */
-    public static final String TYPE = "handledError";
+    public static final String WARNING_TYPE = "handledError";
```

Comment:
This constant was used in this PR #573.
Can you update code from main branch and modify it?
![image](https://github.com/microsoft/HydraLab/assets/26757995/0e329c5d-7658-4022-9245-54bed2b5a5fe)

Reply:
@zhou9584 updated, please help approve it, thanks for the callout.